packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -0,0 +1,54 @@
+﻿# Angular for XMCloud
+
+> Sitecore JSS Angular App for XM Cloud.
+
+[Documentation](<TODO>)
+
+This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. The supported key features are as follows:
+
+- Context ID / envoy proxy
+
+- Pages new editing integration via HTTP render engine endpoint (e.g. editing/render)
+
+- XMCloud proxy personalization
+
+- Forms support
+
+This SPA is tailored to enhance development workflows and enable full utilization of XM Cloud’s capabilities, providing a seamless and efficient foundation for developers.
+
+## Components and Supporting Applications
+
+The following components and supporting applications have been added to the Angular base app to ensure compatibility with XM Cloud:
+
+- `XMCloud Angular`: Adds support for the Sitecore Context data, which simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products. Also includes example components for SXA projects.
+
+- `XMCloud Proxy`: Adds integration with XMCloud for the JSS SPA applications. Considered experimental.
```

Comment:
> Considered experimental.

The whole XMCloud Angular setup is experimental (or 'preview release') for the moment. Let's add this disclaimer at the top instead.